docs/cache/getting-started.mdx

```diff
@@ -0,0 +1,107 @@
+---
+sidebar_position: 3
+sidebar_label: Getting Started
+title: Getting Started
+pagination_prev: null
+pagination_next: null
+description: Jump in and get started quickly using Momento Cache with your SDK of choice.
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Getting started with Momento Cache
+
+If you're looking to jump in and get started with Momento Cache with an SDK, you've come to the right place.
```

Comment:
It seems like this page will offer 3 ways to get started with cache: console, CLI, or SDKs. Probably better to remove "with an SDK" and introduce the options here?

Reply:
good catch!